internal/mosaic/content.go

```diff
@@ -17,9 +17,10 @@ type (
 	}
 
 	Mosaic struct {
-		Name      string  `json:"name"`
-		Medias    []Media `json:"medias"`
-		WithAudio bool    `json:"with_audio"`
+		Name          string  `json:"name"`
+		BackgroundUrl string  `json:"backgroundUrl"`
```

Comment:
A better Golang convention is to use `BackgroundURL`
And for the JSON name field, I would choose `background_url`

Reply:
Alright, i wasn't attentive of this.

Fixed.